### PR TITLE
Reorganize codelab

### PR DESCRIPTION
**Pull request by phbnf**

### CODE REVIEW

File: cmd/conformance/README.md [hunk 1]
```diff
@@ -9,12 +9,21 @@ Implementations are provided that use:
...
+ - [AWS](./aws/)
 
 Each of these personalities exposes an endpoint that accepts `POST` requests at a `/add` URL.
 The contents of any request body will be appended to the log, and the decimal index assigned to this newly _sequenced_ entry will be returned.
 
 ## Codelab
 
+This codelab will help you add a few entries to a log, and read log contents.
+First, you need to bring up a Trillian Tessera log and to start personality (a server that 
+interfaces with the log) on the infrastructure of your choice:
```
Comment (AlCutter) [inline cmd/conformance/README.md:21]:
```suggestion
First, you need to bring up personality (a server built with Tessera which manages the log) on the infrastructure of your choice:
```

File: cmd/conformance/README.md [hunk 2]
```diff
@@ -9,12 +9,21 @@ Implementations are provided that use:
...
+ - [A local filesystem](./posix#bring-up-a-log)
```
Comment (AlCutter) [inline cmd/conformance/README.md:22]:
```suggestion
 - [A local POSIX-compliant filesystem](./posix#bring-up-a-log)
```

File: cmd/conformance/mysql/README.md [hunk 3]
```diff
@@ -1,54 +1,70 @@
-# How to Run a Tessera Log (MySQL)
+# Conformance MySQL log
+
+This binary runs an HTTP web server that accepts POST HTTP requests to a `/add` endpoint.
```
Comment (AlCutter) [inline cmd/conformance/mysql/README.md:3]:
```suggestion
This binary runs an HTTP web server that accepts POST HTTP requests to an `/add` endpoint.
```

File: cmd/conformance/mysql/README.md [hunk 4]
```diff
@@ -1,54 +1,70 @@
...
 > - This is an example and is not fit for production use, but demonstrates a way of using the Tessera Log with MySQL storage backend.
 > - This example is built on the [tlog tiles API](https://c2sp.org/tlog-tiles) for read endpoints and exposes a /add endpoint that allows any POSTed data to be added to the log.
 
-The tessera log with the MySQL storage implementation can be started with either Docker Compose or manual `go run`.
+## Bring up a log
+
+This will help you bring up a MySQL database to store a Tessera log, and start a personality
+binary that can add entries to it.
+
+You can either start this log using Docker Compose, or manually with `go run`.
```
Comment (AlCutter) [inline cmd/conformance/mysql/README.md:15]:
```suggestion
You can run this personality using Docker Compose or manually with `go run`.
```

File: cmd/conformance/mysql/README.md [hunk 5]
```diff
@@ -1,54 +1,70 @@
...
 docker compose -f ./cmd/conformance/mysql/docker/compose.yaml down
 ```
 
-## Manual 
+### Manual 
+
+#### Prerequisites
 
-### Prerequisites
+You need to have a MySQL database running on port `3306`.
```
Comment (AlCutter) [inline cmd/conformance/mysql/README.md:43]: Will the reader also need to have configured the MySQL instance to accept password auth for `root` and set the password to `root`, and created a DB instance called `test_tessera`?

File: cmd/conformance/mysql/README.md [hunk 6]
```diff
@@ -1,54 +1,70 @@
...
+#### Stop the log
 
 <kbd>Ctrl</kbd> <kbd>C</kbd>
 
-## Using the log
+## Add entries to the log
+
+### Manually
+
+Head over to the [codelab](../#codelab) to manually add entries to the log, and read them.
```
Comment (AlCutter) [inline cmd/conformance/mysql/README.md:65]:
```suggestion
Head over to the [codelab](../#codelab) to manually add entries to the log, and read them back.
```

File: cmd/conformance/posix/README.md [hunk 7]
```diff
@@ -1,25 +1,35 @@
 # conformance-posix
-
-This command runs an HTTP web server that accepts POST HTTP requests to a `/add` endpoint.
+This binary runs an HTTP web server that accepts POST HTTP requests to a `/add` endpoint.
```
Comment (AlCutter) [inline cmd/conformance/posix/README.md:2]:
```suggestion
This binary runs an HTTP web server that accepts POST HTTP requests to an `/add` endpoint.
```

File: cmd/conformance/posix/README.md [hunk 8]
```diff
@@ -1,25 +1,35 @@
...
   --storage_dir=${LOG_DIR} \
   --initialise \
   --listen=:2025 \
   --v=2
 ```
 
-In another terminal, run the hammer against the log.
+## Add entries to the log
+### Manually
+Head over to the [codelab](../#codelab) to manually add entries to the log, and read them.
```
Comment (AlCutter) [inline cmd/conformance/posix/README.md:29]:
```suggestion
Head over to the [codelab](../#codelab) to manually add entries to the log, and read them back.
```

**Review by AlCutter — APPROVED** [review]
Just a few nits inline.

File: cmd/conformance/mysql/README.md (hunk 5 above)
Comment (phbnf) [reply cmd/conformance/mysql/README.md:43]: @roger2hk

File: cmd/conformance/mysql/README.md (hunk 6 above)
Comment (phbnf) [reply cmd/conformance/mysql/README.md:65]: used "and inspect the log" to be more accurate.

File: cmd/conformance/posix/README.md (hunk 8 above)
Comment (phbnf) [reply cmd/conformance/posix/README.md:29]: Used "and inspect the log" instead

File: cmd/conformance/mysql/README.md (hunk 5 above)
Comment (roger2hk) [reply cmd/conformance/mysql/README.md:43]: Yes, the "starting the log" command in the next few lines requires the user and password to be `root`. We can turn them into variables, but I think it's easier to define them for easy & paste.

Comment (roger2hk) [reply cmd/conformance/mysql/README.md:43]: If we don't want to use the `root` user, we can define the `MYSQL_DATABASE`, `MYSQL_USER`, `MYSQL_PASSWORD` in the `docker run` command.

Comment (phbnf) [reply cmd/conformance/mysql/README.md]: Alright, added some instructions.

**Pull request merged**